recipe/meta.yaml

```diff
@@ -8,34 +7,41 @@ package:
 source:  # [py27 or py3k]
   # Avro exist as two versions in apache avro for py2 and py3.
   fn: avro-{{ version }}.tar.gz  # [py27]
-  url: {{ mirror }}/avro/avro-{{ version }}/py/avro-{{ version }}.tar.gz  # [py27]
-  sha256: 'e4181a9dc686c82cbc9bb496787d659d55582c44ad74b69924c45f80f3d513a1'  # [py27]
+  url: https://files.pythonhosted.org/packages/a1/9a/a2c83f6105d975ec6680b2cab79b4e3bd51ae33f31c10debe8d2485c8f80/avro-{{ version }}.tar.gz  # [py27]
+  sha256: 9fc06bf0b91877b1b6afac8ad35d66d2e545be2b9b2820e5fba4e35b767bfb9a  # [py27]
 
   fn: avro-python3-{{ version }}.tar.gz  # [py3k]
-  url: {{ mirror }}/avro/avro-{{ version }}/py3/avro-python3-{{ version }}.tar.gz  # [py3k]
-  sha256: 'a3e698f09e702571bc2283471ed97f7055ae4c055a037624d04a95a320242ccf'  # [py3k]
+  url: https://files.pythonhosted.org/packages/76/b2/98a736a31213d3e281a62bcae5572cf297d2546bc429accf36f9ee1604bf/avro-python3-{{ version }}.tar.gz  # [py3k]
+  sha256: daab2cea71b942a1eb57d700d4a729e9d6cd93284d4dd4d65a378b9f958aa0d2  # [py3k]
 
 build:
-  number: 1000
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
-    build:
-        - python
-        - setuptools
-    run:
-        - python
+  build:
```

Comment:
this section should be removed.  we only need host / run

Reply:
I removed it, but I'm curious why it is not required. The example of the `meta.yaml` file [here](https://docs.conda.io/projects/conda-build/en/latest/user-guide/tutorials/build-pkgs.html#editing-the-meta-yaml-file) shows only build and run sections (no host), but the attached example has host and run.